composer.json

```diff
@@ -200,7 +200,10 @@
     "autoload-dev": {
         "psr-4": {
             "Tests\\": "tests/"
-        }
+        },
+        "files": [
+            "tests/functions.php"
```

Comment:
I preferred a separate file than having the function definition in the bootstrap file